resources/js/home.js

```diff
@@ -1,73 +1,93 @@
-import utils from './utils.js';
+import utils from './utils.js'
+import tableFn from "./table.js"
 
-var fromMenuObj
-var indexContentObj
-var paraHeight
+let paraHeight
+let headerObj, contentObj
 
 
-// Loading Contents for Homepage
-const contentRequest = new XMLHttpRequest()
+/*--------------- Home page content loading section ---------------*/
 const para = document.querySelector('#blog-paragraph')
 const imageArea = document.querySelector('.image-area')
 
-contentRequest.onload = function () {
-  if (this.status === 200) {
-    try {
-      indexContentObj = JSON.parse(this.responseText)
-      para.textContent = indexContentObj.Content
-
-      const image = document.createElement('img')
-      image.src = indexContentObj.image
-      image.className += 'content-img';
-      imageArea.appendChild(image)
-
-      paraResize()
-      
-    } catch {
-      console.warn('JSON not parsed')
-    }
-  } else {
-    console.warn('File not found')
-  }
-}
+utils.jsonCaller('get', 'resources/json/homepage.json', function (object) {
+  para.textContent = object.content
+  // let headerObj = object.table
+  headerObj = object.table
```

Comment:
headerObj can be made a local-scoped const here, as it is being passed as args to the subsequent functions

Reply:
unnecessary global declarations removed.